frontend/public/locales/fr/status.json

```diff
@@ -127,7 +127,7 @@
     "exit-btn": "Quitter le vérificateur de l'état",
     "exit-link": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#etat",
     "status-not-found": {
-      "heading": "Nous n'avons pas pu trouver le statut de votre candidature avec les informations que vous nous avez fournies.",
+      "heading": "Nous n'avons pas pu trouver l'état de votre demande avec les informations",
```

Comment:
```suggestion
      "heading": "Nous n'avons pas pu trouver l'état de votre demande avec les informations.",
```

Thoughts?

Reply:
yeah good call.  Then it matches english french